### PR TITLE
fixup some ReSpec issues

### PR DESCRIPTION
**Pull request by marcoscaceres**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -163,10 +106,10 @@ <h2>Definition</h2>
 
       <p>This one <a>selection</a> must be shared by all the content of the <a>document</a>
         (though not by nested <a>document</a>s), including any editing hosts in the document.
-        <dfn>Editing hosts</dfn> (<a href="https://www.w3.org/TR/html5/editing.html#editing-host">defined</a> in [[!HTML5]]) that are not
+        <dfn data-cite="!HTML#editing-host">Editing hosts</dfn> that are not
```
Comment (marcoscaceres) [inline index.html:109]: There are a bunch xrefs in this spec that would benefit from `<dfn data-cite>`. I only did a few, just to show how to do it.

Comment (marcoscaceres) [reply index.html:109]: More info here:
https://github.com/w3c/respec/wiki/data--cite

**Comment by marcoscaceres** [comment]
@rniwa thanks! I've got a few incoming PRs over on ReSpec which will fix some additional things. Will ping back once they get merged.

**Comment by rniwa** [comment]
Okay! Thanks for the much needed fixes!

**Comment by marcoscaceres** [comment]
Ok, spec's a bit happier now.

**Pull request merged**